src/app/pages/OriginsLaunchpad/pages/SalesDay/storage.ts

```diff
@@ -0,0 +1,22 @@
+import { originsSaleStorageKey } from 'utils/classifiers';
```

Comment:
I am not sure if the whole idea of saving the active step to local storage is not overkill - we basically just want our users to be able to refresh without worries and that's done by routes. 

What do you think @soulBit ?

Reply:
I agree with tiltom on this comment, unless there was a specific reason it was added?

Reply:
when users refresh in the 3rd step, the wallet needs to be connected again, and then the routing system will lead the user to the previous page programmatically.
So I think it's still needed.

Reply:
Ok, if you need it there, it can stay the way it is.